.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+source = .
+omit =./tests/*,./env/*,manage.py,config.py
+
+[report]
+omit = ./env/*,./tests/*
```

Comment:
By default all the packages and files are included in coverage stats, this config file makes it so we only include what we wrote in the coverage stats.